docs/io/physical_simulation.rst

```diff
@@ -149,8 +149,10 @@ SiDB Operational Domain
 
         .. doxygenstruct:: fiction::write_operational_domain_params
            :members:
-        .. doxygenfunction:: fiction::write_operational_domain(const operational_domain& opdom, std::ostream& os, const write_operational_domain_params& params = {})
-        .. doxygenfunction:: fiction::write_operational_domain(const operational_domain& opdom, const std::string_view& filename, const write_operational_domain_params& params = {})
+        .. doxygenfunction:: fiction::write_operational_domain(const operational_domain<>& opdom, std::ostream& os,
```

Comment:
@marcelwa Do you have any idea why it does not work? I already tried it without ``<>`` but without success.

Reply:
Thank you! I will try it.